components/Channels.js

```diff
@@ -81,7 +81,7 @@ function Operation({ operation }) {
           ))}
         </>
       ) : (
-        <Message title='Message' message={operation.message(0)} />
+        <Message title={`Message \`${operation.message(0)}\``} message={operation.message(0)} />
```

Comment:
Currently you render whole object and you end with output like:

```md
##### Message `[object Object]`

*This is just a dummy info message*

...
```

you should pass to the title `prop` only `uid` value like:

```suggestion
        <Message title={`Message \`${operation.message(0).uid()}\``} message={operation.message(0)} />
```

and then you will have:

```md
##### Message `dummyInfo`

...
```